spec/models/article_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'DB columns' do
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :content }
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:article)).to be_valid
```

Comment:
Please use the shorter `create(:article)`

Reply:
I am not quite sure what this means, could you elaborate a little on what a shorter ```
create(:article)
``` is

Reply:
I’ve demonstrated several times how we can include FactoryBot syntax methods to your rails_helper and use this shortened syntax in your specs.